client/signup/config/steps-pure.js

```diff
@@ -1,6 +1,7 @@
 /**
  * External dependencies
  */
+import React from 'react';
```

Comment:
Is this needed here?

Reply:
I believe it is. There are a couple of instances of react components being used in the file: [here](https://github.com/Automattic/wp-calypso/pull/43061/files/2ead16c2756993077a34fb59c032a1c8df6e28f5#diff-ae0bf020974ae29c034739f287afc5c5R175) and [here](https://github.com/Automattic/wp-calypso/pull/43061/files/2ead16c2756993077a34fb59c032a1c8df6e28f5#diff-ae0bf020974ae29c034739f287afc5c5R185). 
Because of this, I remember the build failing without the React import.

Reply:
Ah okay, sounds good.